Generators/src/main/java/io/deephaven/pythonPreambles/plotV2.py

```diff
@@ -104,9 +104,9 @@ class Figure(JObjectWrapper):
 
     j_object_type = jpy.get_type("io.deephaven.plot.Figure")
 
-    def __init__(self, j_figure: jpy.JType = None):
+    def __init__(self, rows: int = 1, cols: int = 1, j_figure: jpy.JType = None):
         if not j_figure:
-            self.j_figure = _JPlottingConvenience.figure()
+            self.j_figure = _JPlottingConvenience.figure(rows, cols)
         else:
             self.j_figure = j_figure
```

Comment:
It seems like we should throw an error if `j_figure` plus either `rows` or `cols` is passed in.

Reply:
When using default values (rows=1, cols=1), we can't easily tell if the user specified `Figure()` vs. `Figure(1,1)`, so wouldn't be able to accurately raise an error.